src/Nova/Order.php

```diff
@@ -26,6 +26,11 @@ class Order extends Resource
         'order_number',
     ];
 
+    /** @psalm-suppress UndefinedClass */
+    protected array $filterClassList = [
+        \Tipoff\Locations\Nova\Filters\Location::class,
+    ];
```

Comment:
The base class introduces arrays for filters, lenses, actions and cards.  The actual resource class fill those arrays with class **names** (not instances) for the things that could be supported if they exist.  When actually responding the the nova `filters(...)` method, the base class runs through the array, eliminates classes that don't exist and instantiates those that do.

Reply:
This is a great solution!